src/magic_folder/test/cli/test_magic_folder.py

```diff
@@ -50,9 +50,8 @@
 from ...frontends.magic_folder import (
     MagicFolder,
 )
-from ...scripts import (
-    magic_folder_cli,
-)
+from magic_folder import cli as magic_folder_cli
```

Comment:
explicit relative import here would be more consistent with surrounding code, not that I'm a huge fan of the dot counting that goes into understanding something like `from ...frontends.magic_folder import ...`